server/webapp/WEB-INF/rails.new/lib/extensions/java_util_date.rb

```diff
@@ -27,4 +31,5 @@ def to_long_display_date_time
     pattern = java.text.SimpleDateFormat.new("dd MMM, yyyy 'at' HH:mm:ss [Z]" )
     pattern.format(self)
   end
+
```

Comment:
I know we removed `to_json` on `java.util.Date`. This may likely break existing APIs (user, backup). Can we bring this back until we call `.utc` on dates in those APIs?

Alternatively, can we implement `as_json` as follows - 

```ruby
def as_json(options={})
    self.utc.as_json(options)
end
```